docs/quickstart.rst

```diff
@@ -12,7 +12,7 @@ Let’s build a basic GraphQL schema from scratch.
 Requirements
 ------------
 
--  Python (2.7, 3.4, 3.5, 3.6, pypy)
+-  Python (3.6+, pypy)
```

Comment:
3.5 should be supported too (until 3.8 is released in October).

Reply:
I guess we might want to clarify the versions supported and the plan in ROADMAP.md

Reply:
I went with 3.6+ after looking at what we're testing for in CI. This is absolutely open to discussion!